src/pages/ScreenRouting/mediaScreens/MediaEpisode/MediaEpisode.tsx

```diff
@@ -0,0 +1,46 @@
+import { Navigate } from 'react-router';
+import { useTranslation } from 'react-i18next';
+
+import type { ScreenComponent } from '#types/screens';
+import type { PlaylistItem } from '#types/playlist';
+import Loading from '#src/pages/Loading/Loading';
+import { deprecatedSeriesURL, mediaURL } from '#src/utils/formatting';
+import useQueryParam from '#src/hooks/useQueryParam';
+import useGetSeriesId from '#src/hooks/series/useSeriesId';
+import ErrorPage from '#components/ErrorPage/ErrorPage';
+import { useSeriesData } from '#src/hooks/series/useSeriesData';
+
+/**
+ * This media series episode screen is used to redirect an episode item to the series page.
+ */
+const MediaEpisode: ScreenComponent<PlaylistItem> = ({ data: media, isLoading: isMediaLoading }) => {
+  const mediaId = media.mediaid;
+
+  const { t } = useTranslation('video');
+  const play = useQueryParam('play') === '1';
+  const feedId = useQueryParam('r');
+
+  const { deprecatedPlaylistId, seriesId, isLoading: isSeriesIdLoading } = useGetSeriesId(media);
```

Comment:
Suggest you remove the legacy / deprecated support from useGetSeriesId and do that part only on the legacy series page (conditionally lookup series ID and append to the URL if it doesn't exist.)

Reply:
I did it for MediaSeries page to remove the container, but for the episode page we still need to know the parent series media item to redirect to the MediaSeries page or playlist Id for the LegacySeries page. Have not found a good way of doing it other way yet.

Reply:
Maybe I found a possible solution of using either series id retrieved from custom params or tags to redirect to Legacy page. You can have a look